mrmustard/lab_dev/states/base.py

```diff
@@ -689,6 +718,41 @@ def from_phase_space(
         ret._representation = Bargmann(*wigner_to_bargmann_rho(cov, means))
         return ret
 
+    @classmethod
+    def from_quadrature(
+        cls,
+        modes: Sequence[int],
+        triple: tuple[ComplexMatrix, ComplexVector, complex],
+        name: Optional[str] = None,
+    ) -> DM:
+        QtoBMap_CC = _BtoQMap(modes).dual.adjoint @ _BtoQMap(modes).dual
```

Comment:
The logic in this func is very hard to follow. I would add inline comments to explain to posterity what is going on